### ISSUE TITLE
UNARYTEST: belong to enum

### ISSUE THREAD

**Issue by Bab64** [al66/imicros-feel-interpreter#2]
Hi. Just simple question: var. `enum` comes from a DMN table. I just want to evaluate the (true) fact that A belongs to `enum`. 2nd ligne of code failed by requesting UNARYTESTS function (which is the case). In short, how to write this with imicros-feel-interpreter? Thanks.
```
const enum = '"A","B"';
const result = interpreter.evaluate(enum,{"?": "A"});
```

Test case:
[https://nikku.github.io/feel-playground/?e=%22A%22%2C%22B%22&c=%7B%0A++%22%3F%22%3A+%22A%22%0A%7D&t=unaryTests&st=true](https://nikku.github.io/feel-playground/?e=%22A%22%2C%22B%22&c=%7B%0A++%22%3F%22%3A+%22A%22%0A%7D&t=unaryTests&st=true)

**Comment by al66** [comment]
The unary test can be evaluated with ```value in (condition)```

You can write your test as:
```
const cell = '"A","B"';
const result = interpreter.evaluate('a in (' + cell + ')',{a: "A"});
```

**Comment by Bab64** [comment]
Thanks. This works...

**Comment by Bab64** [comment]
Testing existing DMN files, I observe tiny problems ("doesn't work?"):
interpreter.evaluate('? in (' + '"A"' + ')',{'?': "A"}); // Could be replaced by equality test, but it's just singleton set "belongs to"
interpreter.evaluate('? in (' + '"A","B"' + ')',{'?': "-"}); // Hyphen meaning "any"...

**Comment by al66** [comment]
Fixed with version 0.0.9
```
        it("should evaluate 5 in - -> true", () => {
            let result = interpreter.evaluate("5 in -");
            expect(result).toEqual(true);
        });
        it("should evaluate 5 in (-) -> true", () => {
            let result = interpreter.evaluate("5 in (-)");
            expect(result).toEqual(true);
        });

```
